stable/golang/kubeless.tpl.go

```diff
@@ -21,12 +21,10 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
-
```

Comment:
leave these empty lines, they are used to differentiate between core libraries and custom ones

Reply:
right, sorry, will fix this.